app/frontend/src/pages/ask/Ask.tsx

```
@@ -186,6 +195,19 @@ export function Component(): JSX.Element {
             case "includeCategory":
                 setIncludeCategory(value);
                 break;
+            case "sortBy":
+                const newSortBy = value as string;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)